l3kernel/testfiles/m3text006.lvt

```diff
@@ -0,0 +1,141 @@
+%
+% Copyright (C) 2022 The LaTeX Project
+%
+\input{regression-test}
+
+\RequirePackage[enable-debug]{expl3}
+\ExplSyntaxOn
+\debug_on:n { check-declarations , deprecation , log-functions }
+\ExplSyntaxOff
+
+\documentclass{minimal}
+
+\START
+\AUTHOR{Joseph Wright}
+
+\ExplSyntaxOn
+
+\TEST { Grapheme~mapping }
+  {
+    \OMIT
+      \ior_open:Nn \g_tmpa_ior { GraphemeBreakTest.txt }
```

Comment:
That test file is a bit heavy without any documentation what it does or supposed to be tested. Could there be a few words up at the top  saying that?